knuverse/knufactor.py

```diff
@@ -200,26 +200,30 @@ def auth_token(self, apikey, secret):
         return self._create_response(response).get("jwt")
 
     @_auth
-    def auth_grant(self, client):
+    def auth_grant(self, client, role=None, mode=None):
         """
         Used to get a grant token.  Grant tokens expire after 1 minute.
         Grant tokens can be used to start verifications.  Uses POST to /auth/grant interface
 
         :Args:
           * *client*: (str) Client name
+          * *role*: (str or None) The grant token role. Can be "grant_verify", "grant_enroll", or "grant_enroll_verify". If role is not sent in, the role defaults to "grant_verify".
+          * *mode*: (str or None) The mode to perform actions with. Can be "audiopass" or "audiopin". It defaults to the module setting's "mode_default" if None is passed in.
```

Comment:
Could you move the two optional args(role and mode) to a :Kwargs: section? You can look elsewhere in this file for examples.